### ISSUE TITLE
Transcription var error with http request mapped vars

### ISSUE THREAD

**Issue by malconvsilva** [baptisteArno/typebot.io#1664]
I'm trying to use the chat transcription variable, but it doesn't work correctly when I use an http request with more than one variable mapped.

When trying to use transcription and in the flow there is an http request that returns multiple values and fills more than one variable, only the first one is filled in the transcription and if there is a conditional depending on the other variables, the transcription will fail in the conditional and will be different from the flow normal for the bot.

Typebot version 2.27.0 (Self-hosted)

**Comment by baptisteArno** [comment]
Can you clearly describe it by following this template:

1. Steps to reproduce
2. What's happening
3. What were you expecting

It's even better if you can provide a screen recording.

This ultimately allow me to reproduce the bug so that I can fix it properly.

**Comment by malconvsilva** [comment]
Print:
<img width="767" alt="1" src="https://github.com/user-attachments/assets/531e4b3d-dd97-47d0-b063-df6f8c2d79e1">

Flow:
[typebot-export-meu-typebot-ohb064z.json](https://github.com/user-attachments/files/16379981/typebot-export-meu-typebot-ohb064z.json)

What is happening is that if there is an http request in the flow, the transcription ignores the variables obtained from the http request, and if there is a conditional depending on these variables, the transcription follows the wrong flow.

I created a flow that asks some questions and http requests and at the end sends the transcript to the user, but it wasn't working and after some tests I discovered that the failure occurred in http requests (I tested http and nocodb node).
I made a simpler flow, with an http request that fills two variables, and a condition if the second variable is filled follows one flow, otherwise it follows another.
